src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,53 @@
+package duke;
+
+import java.io.*;
```

Comment:
Should the imports be declared explicitly instead of using wildcard? Also noticed in a few other files